script.js

```diff
@@ -1,4 +1,312 @@
+  playerScore = calculateScore(playerHand);
+
+  // Draws 2 cards for computer
+  var computerDraws = drawCard(2, computerHand);
+  displayComputerHand = displayCard(computerHand);
+  computerScore = calculateScore(computerHand);
+
+  console.log("Player's first hand", playerHand);
+  console.log("Computer's first hand", computerHand);
+  console.log("First hand, player's score", playerScore);
+  console.log("First hand, computer's score", computerScore);
+
+  // Message
+  var output = document.getElementById("output-div");
+
+  if (playerScore < 21 && computerScore < 21) {
+    output.innerHTML = `1st round <br><br> Your hand: ${displayPlayerHand} <br> Score: ${playerScore} <br><br> Computer's hand: ${displayComputerHand[0]}, second card (face down). <br><br> Number of cards in deck: ${deck.length} <br><br> To continue, press Hit or Stand.`;
+  } else {
+    output.innerHTML = `1st round <br><br> ${clearOutcomes()}`;
+  }
+});
+
+// Hit button
+var hitButton = document.getElementById("hit-button");
+hitButton.addEventListener("click", function () {
+  var playerDraws = drawCard(1, playerHand);
+  displayPlayerHand = displayCard(playerHand);
+  playerScore = calculateScore(playerHand);
+
+  // Message
+  var output = document.getElementById("output-div");
+
+  if (playerScore < 21 && computerScore < 21) {
+    output.innerHTML = `Your hand: ${displayPlayerHand} <br> Score: ${playerScore} <br><br> Computer's hand: ${displayComputerHand[0]}, second card (face down). <br><br> Number of cards in deck: ${deck.length} <br><br> To continue, press Hit or Stand.`;
+  } else {
+    output.innerHTML = clearOutcomes();
+  }
+});
+
+// Stand button
+var hitButton = document.getElementById("stand-button");
+hitButton.addEventListener("click", function () {
+  var output = document.getElementById("output-div");
+  if (computerScore < 17) {
+    while (computerScore < 17) {
+      console.log("Computer's hand", computerHand);
+      console.log("Computer's score", computerScore);
+      var computerDraws = drawCard(1, computerHand);
+      displayComputerHand = displayCard(computerHand);
+      computerScore = calculateScore(computerHand);
+    }
+    if (computerScore < 21 && playerScore < 21) {
+      output.innerHTML = determineWinner();
+    } else {
+      output.innerHTML = clearOutcomes();
+    }
+  } else {
+    if (computerScore < 21 && playerScore < 21) {
+      output.innerHTML = determineWinner();
+    } else {
+      output.innerHTML = clearOutcomes();
+    }
+  }
+});
+
+//Reset Game
+var resetButton = document.getElementById("reset-button");
+resetButton.addEventListener("click", function () {
+  deck = [];
+  playerHand = [];
+  computerHand = [];
+  displayPlayerHand = [];
+  displayComputerHand = [];
+
+  var output = document.getElementById("output-div");
+  output.innerText = `The game has been reset. Press Start to play a new game.`;
+});
+
+// Create a deck of cards
+var makeDeck = function () {
```

Comment:
Similar to how you grouped all your global variables, try to group all your functions together as well